### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by nfishel48**
A sample test for the django testing framework

### CODE REVIEW

File: core/tests.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 from django.test import TestCase
-
+from core.models import *
 # Create your tests here.
```
Comment (SuprajaKalva) [inline core/tests.py:5]: Are all the tests for the entire project included here? Or are there multiple files?

**Comment by SuprajaKalva** [comment]
Other than the comment above, I think it looks good!

**Comment by derickso8** [comment]
Nice work! Everything looks fine to me.

**Comment by Aadamko1** [comment]
Looks good to me

**Comment by ghost** [comment]
Looks good to me

**Review by ghost — COMMENTED** [review]
looks good

**Review by ghost — COMMENTED** [review]
looks good

**Pull request merged**